### PR TITLE
Track mattermost repo structure in dependencies

### PR DESCRIPTION
**Pull request by i4h**
I was unable to run the bot because the required gitlab repo did not exist (Unfortunately go get doesnt really tell you whats wrong...)
This fixed it.

### CODE REVIEW

File: bot_sample.go [hunk 1]
```diff
@@ -8,8 +8,7 @@ import (
 	"os/signal"
 	"regexp"
 	"strings"
-
-	"github.com/mattermost/platform/model"
+	"github.com/mattermost/mattermost-server/model"
```
Comment (MusikPolice) [inline bot_sample.go]: I suspect that this is the only necessary change to solve the problem. We split the old `mattermost/platform` repository into `mattermost/mattermost-server` and `mattermost/mattermost-webapp` awhile back, and forgot to update this sample project.

The other changes in these files are likely unnecessary, and should be reverted.

Comment (i4h) [reply bot_sample.go]: you are right, only one of the two commits should have been in the request. i will change.
however, the change in glide.yaml should be merged too, yes?

Comment (i4h) [reply bot_sample.go]: i created a new merge request containing only the one commit

Comment (MusikPolice) [reply bot_sample.go]: Ok, I'm going to close this request then. Thanks for the submission!

**Pull request closed without merging**